rllib/agents/a3c/a3c_torch_policy.py

```diff
@@ -38,19 +38,21 @@ def add_advantages(policy,
                    sample_batch,
                    other_agent_batches=None,
                    episode=None):
-    completed = sample_batch[SampleBatch.DONES][-1]
-    if completed:
+
+    if sample_batch[SampleBatch.DONES][-1]:
```

Comment:
Prefer to use intermediate variables to clarify the value of a long expression when psosible.